openstack/manila/templates/seeds.yaml

```diff
@@ -64,6 +64,8 @@ spec:
   - name: cloud_sharedfilesystem_viewer
   - name: sharedfilesystem_admin
   - name: sharedfilesystem_viewer
+{{- Role `cloud_objectstore_admin` needed for backup to cloud service }}
```

Comment:
This is not a valid format for a comment, maybe add it to the role assignment L118 below at the end after putting `# `